input/fsh/modules/biospecimen.fsh

```diff
@@ -85,15 +85,22 @@ Description: "Concentration of the Aliquot"
 * value[x] only Quantity
 * valueQuantity ^short = "Specify the concentration of the aliquot"
 
-/*Combined Profile*/
 
+/* Invariant to require collection for parent samples*/
+Invariant:   collection-parent
+Description: "If collection.method is not unknown, then there should be no parent sample"
+Expression:  "parent.empty() implies collection.method.exists()"
```

Comment:
Should we make this `collection.exists` ? I'm not sure it has to be the method, but that an "unknown method" is one way to ensure there is a method.